src/executor/evaluate/mod.rs

```diff
@@ -310,5 +310,51 @@ async fn evaluate_function<'a, T: 'static + Debug>(
             Nullable::Null => Ok(Value::Null),
         }
         .map(Evaluated::from),
+        Function::Div { dividend, divisor } | Function::Mod { dividend, divisor } => {
+            let name = if matches!(func, Function::Div { .. }) {
+                "DIV"
+            } else {
+                "MOD"
+            };
+
+            let dividend = match eval(dividend).await?.try_into()? {
+                Value::F64(number) => number,
+                Value::I64(number) => number as f64,
+                Value::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+                _ => {
+                    return Err(EvaluateError::FunctionRequiresFloatOrIntegerValue(
+                        name.to_owned(),
+                    )
+                    .into());
+                }
+            };
+
+            let divisor = match eval(divisor).await?.try_into()? {
+                Value::F64(number) => match number {
+                    x if x == 0.0 => return Err(EvaluateError::InvalidDivisorZero.into()),
+                    _ => number,
+                },
+                Value::I64(number) => match number {
+                    0 => return Err(EvaluateError::InvalidDivisorZero.into()),
+                    _ => number as f64,
+                },
```

Comment:
btw handling divide-by-zero error is so nice, it is even not handled by `Value` itself. thanks 👍